xml/ay_bigfile.xml

```diff
@@ -8958,6 +8958,68 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
 &lt;/users&gt;</screen>
     </example>
 
+    <para>
+     The next example shows a more complete example. The data in
+     <filename>/etc/default/useradd</filename> is used to determine the home
+     directory of the user to be created plus other parameters.
+    </para>
+
+    <example>
+     <title>Complex User Configuration</title>
+     <screen>&lt;users config:type="list"&gt;
+  &lt;user&gt;
+    &lt;username&gt;root&lt;/username&gt;
+    &lt;user_password&gt;password&lt;/user_password&gt;
+    &lt;uid&gt;1001&lt;/uid&gt;
+    &lt;gid&gt;100&lt;/gid&gt;
+    &lt;encrypted config:type="boolean"&gt;false&lt;/encrypted&gt;
+    &lt;fullname&gt;Root User&lt;fullname&gt;
+    &lt;authorized_keys config:type="list"&gt;
+      &lt;listentry&gt;command="/opt/login.sh" ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDKLt1vnW2vTJpBp3VK91rFsBvpY97NljsVLdgUrlPbZ/L51FerQQ+djQ/ivDASQjO+567nMGqfYGFA/De1EGMMEoeShza67qjNi14L1HBGgVojaNajMR/NI2d1kDyvsgRy7D7FT5UGGUNT0dlcSD3b85zwgHeYLidgcGIoKeRi7HpVDOOTyhwUv4sq3ubrPCWARgPeOLdVFa9clC8PTZdxSeKp4jpNjIHEyREPin2Un1luCIPWrOYyym7aRJEPopCEqBA9HvfwpbuwBI5F0uIWZgSQLfpwW86599fBo/PvMDa96DpxH1VlzJlAIHQsMkMHbsCazPNC0++Kp5ZVERiH root@example.net&lt;/listentry&gt;
+    &lt;/authorized_keys&gt;
+  &lt;/user&gt;
+  &lt;user&gt;
+    &lt;username&gt;&exampleuser_plain;&lt;/username&gt;
+    &lt;user_password&gt;password&lt;/user_password&gt;
+    &lt;uid&gt;1002&lt;/uid&gt;
+    &lt;gid&gt;100&lt;/gid&gt;
+    &lt;encrypted config:type="boolean"&gt;false&lt;/encrypted&gt;
+    &lt;fullname&gt;Plain User&lt;fullname&gt;
+    &lt;home&gt;/Users/plain&lt;home&gt;
+    &lt;password_settings&gt;
+      &lt;max&gt;120&lt;/max&gt;
+      &lt;inact&gt;5&lt;/inact&gt;
+    &lt;/password_settings&gt;
+  &lt;/user&gt;
+&lt;/users&gt;</screen>
+    </example>
+
+    <note>
+     <title><literal>authorized_keys</literal> file will be overwritten</title>
+     <para>
+      If the profile defines a set of SSH authorized keys for a user, the
+      content of the <literal>$HOME/.ssh/authorized_keys</literal> file will be
+      overwritten when using AutoYaST to perform a system upgrade or
+      configuration.
+     </para>
+     <para>
+      To avoid such behavior, the <literal>&lt;authorized_keys&gt;</literal> section
+      should be omitted.
+     </para>
+    </note>
+
+    <note>
+     <title><literal>uid</literal> should be be a unique</title>
+     <para>
+      Either each user <literal>uid</literal> must be defined and should be
+      unique, or none <literal>uid</literal> will be defined in the &ay;
+      control file. In that case &ay; evalutes the UIDs
+      of each user. Otherwise it could be that UIDs are not unique on the
+      system which is not an error in general but could cause some problems
+      in other areas.
+     </para>
```

Comment:
hm, not exactly. The problem is if there are entries with IDs and there are entries without IDs too. So YaST evaluates IDs for the not defined entries automatically which can be the same as the IDs which have already been defined in other entries.
So mixing user entries which have and have not IDs is not an good idea :-)